.doc_gen/metadata/services.yaml

```diff
@@ -10,6 +10,11 @@ application-autoscaling:
   short: "&APP-AS;"
   sort: "Application Auto Scaling"
   version: application-autoscaling-2016-02-06
+autoscaling:
+  long: "&AWS-ASlong;"
+  short: "&AWS-AS;"
+  sort: "EC2 Auto Scaling"
+  version: autoscaling-2011-01-01  
```

Comment:
Assuming you do mean EC2 Auto Scaling (and not "AWS Auto Scaling, apparently a different service), this block already exists in this file (under 'auto-scaling', the Smithy name for the service).

Reply:
I'll remove this block.

Reply:
Removed.